client/components/parts/HeroImageField.jsx

```diff
@@ -1,82 +1,66 @@
-import React, {Component, PropTypes} from 'react'
+import React, {PureComponent, PropTypes} from 'react'
+import classNames from 'classnames'
+
+import Icon from 'parts/Icon'
+
 import ImageSearchField from 'parts/ImageSearchField'
 import ImageUploadField from 'parts/ImageUploadField'
-import classNames from 'classnames'
 
-export default class HeroImageField extends Component{
+export default class HeroImageField extends PureComponent{
   constructor(props){
     super(props)
     this.state = {
-      tabStates: this.getInitialTabStates()
+      activeTab: 0
```

Comment:
tabSates used to be an array of strings with true/false correct? (or at least something similar).
Now active tab is a number - how does each number map to a tab name?